setup.py

```diff
@@ -45,19 +45,24 @@
     include_package_data=True,
     zip_safe=False,
     install_requires=[
-        'Flask==1.1.2',
```

Comment:
Why delete all of the library version tags? It's important to pin the libraries down so a Docker build doesn't break functionality.